src/sources/id/indowebnovel.js

```diff
@@ -78,22 +84,23 @@ const parseNovelAndChapters = async novelUrl => {
 
   let chapters = [];
 
-  loadedCheerio('.series-chapterlist li').each(function () {
-    const chapterName = loadedCheerio(this)
-      .find('a span')
-      .first()
-      .text()
-      .replace(/.*?(Chapter.|[0-9])/g, '$1')
+  loadedCheerio('.series-chapterlist li a').each(function () {
+    let titles = loadedCheerio(this)
+      .attr('title')
       .replace(/Bahasa Indonesia/g, '')
-      .replace(/\s+/g, ' ')
+      .replace(/\s\s+/g, ' ')
       .trim();
 
-    const releaseDate = loadedCheerio(this)
-      .find('a span')
-      .first()
-      .next()
-      .text();
-    const chapterUrl = loadedCheerio(this).find('a').attr('href');
+    let titlename = titles
+      .split(' ')
+      .map(title => {
+        return title[0].toUpperCase() + title.slice(1);
+      })
+      .join(' ');
```

Comment:
You could use `startcase` from lodash if you want to capitalize each word on a string.

Reply:
Lodash not available in plugins repo soooo...

Idk how to add 😇

Reply:
It's available in this branch and it'd be easier to maintain.

In plugins, you could create a util function for it instead of repeating.

Reply:
i added a startCase
i think